Logger/LogFactory.cs

```diff
@@ -1,11 +1,36 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public class LogFactory
     {
+        private string path;
+
         public BaseLogger CreateLogger(string className)
         {
-            
-            return null;
+            if (path is null)
+            {
+                return null;
+            } else
+            {
+                BaseLogger logger = new FileLogger(path)
+                {
+                    LoggerName = className
+                };
+
+                return logger;
+            }
+        }
+
+        public void ConfigureFileLogger(string path)
+        {
+            if (path is null || path == "")
+            {
+                throw new ArgumentNullException();
```

Comment:
When throwing an ArgumentNullException pass the name of the argument that was null.
```suggestion
                throw new ArgumentNullException(nameof(path));
```